kafka-observer/run.sh

```diff
@@ -0,0 +1,79 @@
+    exit 1
+fi
+
+if [ "$ES_SERVICE_INSTANCE" == "foobar" ]; then
+    echo "[ERROR] ES_SERVICE_INSTANCE mandatory env var is not defined."
+    exit 1
+fi
+
+# Verify that the ES service instance exists under the same account
+if ! ibmcloud resource service-instance $ES_SERVICE_INSTANCE -q > /dev/null 2>&1; then
+    echo "[ERROR] $ES_SERVICE_INSTANCE service instance does not exists, please ensure the ES instance is created"
+    exit 1
+fi
+
+# clean removes created CE resources
+function clean() {
+  ibmcloud ce configmap delete --name $CM_TOPICJOBS_NAME -f --ignore-not-found
+  ibmcloud ce secret delete --name $AUTH_SECRETS_NAME -f --ignore-not-found
+  ibmcloud ce job delete --name $CONSUMER_JOB_NAME_1 -f --ignore-not-found
+  ibmcloud ce job delete --name $CONSUMER_JOB_NAME_2 -f --ignore-not-found
+  ibmcloud ce job delete --name $OBSERVER_JOB_NAME -f --ignore-not-found
+  ibmcloud ce jobrun delete --name $OBSERVER_JR_NAME -f --ignore-not-found
+}
+
+if [ $# -ge 1 ] && [ -n "$1" ] && [ "$1" == "clean" ]
+then
+    echo "[INFO] going to clean existing project resources"
+    clean
+    exit 0
+fi
+
+# Create CE Resources
+echo "[INFO] Going to create secret ${AUTH_SECRETS_NAME}"
+
+ibmcloud ce secret create --name $AUTH_SECRETS_NAME --from-literal IAM_API_KEY=$IAM_API_KEY
+
+echo "[INFO] Going to create configmap ${CM_TOPICJOBS_NAME}"
+ibmcloud ce configmap create --name $CM_TOPICJOBS_NAME --from-file ${BASEDIR}/resources/kafkadata
+
+echo "[INFO] Going to create JobDefinition ${CONSUMER_JOB_NAME_1}"
+ibmcloud ce job create --name $CONSUMER_JOB_NAME_1 --mode daemon --build-source ${BASEDIR} --build-dockerfile ${BASEDIR}/Dockerfile.consumer --env-from-secret $AUTH_SECRETS_NAME --env-from-configmap $CM_TOPICJOBS_NAME --env CONSUMER_GROUP=payment-consumer-group --env KAFKA_TOPIC=payments --env CE_REMOVE_COMPLETED_JOBS=IMMEDIATELY --wait
+
+echo "[INFO] Going to create JobDefinition ${CONSUMER_JOB_NAME_2}"
+ibmcloud ce job create --name $CONSUMER_JOB_NAME_2 --mode daemon --build-source ${BASEDIR} --build-dockerfile ${BASEDIR}/Dockerfile.consumer --env-from-secret $AUTH_SECRETS_NAME --env-from-configmap $CM_TOPICJOBS_NAME --env CONSUMER_GROUP=shipping-consumer-group --env KAFKA_TOPIC=shipping --env CE_REMOVE_COMPLETED_JOBS=IMMEDIATELY --wait
+
+echo "[INFO] Going to create JobDefinition ${OBSERVER_JOB_NAME}"
+ibmcloud ce job create --name $OBSERVER_JOB_NAME --mode daemon --build-source ${BASEDIR} --build-dockerfile ${BASEDIR}/Dockerfile.observer --env-from-secret $AUTH_SECRETS_NAME --env-from-configmap $CM_TOPICJOBS_NAME --wait
+
+echo "[INFO] Going to bind all JobDefinitions to the provided Event Streams service instance"
+ibmcloud ce job bind --name $CONSUMER_JOB_NAME_1 --service-instance $ES_SERVICE_INSTANCE
+ibmcloud ce job bind --name $CONSUMER_JOB_NAME_2 --service-instance $ES_SERVICE_INSTANCE
+ibmcloud ce job bind --name $OBSERVER_JOB_NAME --service-instance $ES_SERVICE_INSTANCE
```

Comment:
The unbind operations fail if the ES instance name contains spaces, like '`Event Streams-ki`'

Reply:
```suggestion
ibmcloud ce job bind --name $CONSUMER_JOB_NAME_1 --service-instance "$ES_SERVICE_INSTANCE"
ibmcloud ce job bind --name $CONSUMER_JOB_NAME_2 --service-instance "$ES_SERVICE_INSTANCE"
ibmcloud ce job bind --name $OBSERVER_JOB_NAME --service-instance "$ES_SERVICE_INSTANCE"
```